### DISCUSSION TITLE
zsh: segmentation fault (core dumped)

### DISCUSSION THREAD

**Discussion by gtnbssn** [LazyVim/LazyVim#404] [General]
Not sure if I should file an issue yet, but since I have setup lazyvim, nvim crashes from time to time with a segfault. This is not something I had ever had before. This usually happens when I quit. (Only been using lazyvim for 3 days so still need to experiment quite a bit!)

Anyone else affected? Any ideas?

**Comment by folke** [comment]
Indeed not a LAzyVim issue.

First check if the segfault disappears when using Neovim Nightly.

If not, then generate a stack trace (check the Neovim wiki) and report it in the Neovim repo. (unless there is already a similar issue reported)

**Reply by gtnbssn** [reply]
Thanks! I'll look into this.
